File: 2018-12-25/1193/1193.c
#include <stdio.h>

int main(N){
	scanf("%d", &N);
	int floor=1;

	while(N-floor>0){
		N-=floor;
		floor+=1;
	}

	if(floor%2==0)
		printf("%d/%d", N, floor-N+1);
	else	
		printf("%d/%d", floor-N+1, N);
}

File: 2018-12-25/1193/1193.py
from sys import stdin

N = int(stdin.readline())
floor = 1;

while (N-floor>0):
	N-=floor
	floor+=1

if floor%2==0:
	print("%d/%d" %( N,floor-N+1))
else:
	print("%d/%d" %(floor-N+1, N))